CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added `Output` suffix to `Preds`, `FiftyOneDetectionLabels`, `SegmentationLabels`, `FiftyOneDetectionLabels`, `DetectionLabels`, `Classes`, `FiftyOneLabels`, `Labels`, `Logits`, `Probabilities` ([#1011](https://github.com/PyTorchLightning/lightning-flash/pull/1011))
 
+
+- Changed `LabelStudioInput` to rely on the new `Input API` ([#1016](https://github.com/PyTorchLightning/lightning-flash/pull/1016))
+
```

Comment:
I think we can omit the changelog update here as this change is invisible to users
```suggestion
```